.github/workflows/ci.yml

```
@@ -0,0 +1,82 @@
+    - name: Run linting
+      run: npm run lint
+
+    - name: Check formatting
+      run: npm run format:check
+
+    - name: Run tests
+      run: npm run test:coverage
+
+    - name: Upload coverage to Codecov
+      if: matrix.node-version == 18
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)